frontend/src/app/globals.css

```
@@ -123,8 +129,21 @@
   }
   body {
     @apply bg-background text-foreground;
+    background-image:
+      radial-gradient(circle at top left, color-mix(in oklch, var(--primary) 12%, transparent), transparent 34rem),
+      linear-gradient(180deg, color-mix(in oklch, var(--background) 86%, white), var(--background));
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)